x-pack/plugins/xpack_main/server/lib/telemetry/monitoring/get_high_level_stats.js

```diff
@@ -94,6 +102,11 @@ function groupInstancesByCluster(instances, product) {
 
       incrementByKey(cluster.versions, version);
       reduceCloudForCluster(cluster.cloudMap, cloud);
+
+      if (os) {
+        incrementByKey(cluster.os.platform, os.platform);
+        incrementByKey(cluster.os.distro, os.distro);
```

Comment:
Have you tested this when the function is processing Logstash and Beats info? LMK if you have questions about how to do that